applications/app/views/videoEmbed.scala.html

```diff
@@ -103,7 +103,6 @@
         }
 
         document.documentElement.className = docClass;
-        @Html(common.Assets.js.omnitureJs)
```

Comment:
I removed calling omniture from this page in https://github.com/guardian/frontend/pull/14731. Forgot to remove the js from the template though. 👍